src/main/java/uk/gov/ida/verifyserviceprovider/configuration/VerifyHubConfiguration.java

```diff
@@ -2,36 +2,26 @@
 
 import com.fasterxml.jackson.annotation.JsonCreator;
 import com.fasterxml.jackson.annotation.JsonProperty;
+import uk.gov.ida.saml.metadata.MetadataResolverConfiguration;
 
 import java.net.URI;
 
-import static java.util.Optional.ofNullable;
-
 public class VerifyHubConfiguration {
 
-    private URI hubSsoLocation;
-    private HubMetadataConfiguration hubMetadataConfiguration;
+    private final HubEnvironment hubEnvironment;
 
     @JsonCreator
     public VerifyHubConfiguration(
-        @JsonProperty("environment") HubEnvironment hubEnvironment,
-        @JsonProperty("ssoLocation") URI hubSsoLocation,
-        @JsonProperty("metadata") HubMetadataConfiguration hubMetadataConfiguration
+        @JsonProperty("environment") HubEnvironment hubEnvironment
```

Comment:
Is ` local-config.yml ` file under `local-running` directory in use? If yes, can we update the file to reflect these changes since the config file is still using `verifyHubConfiguration.ssoLocation` and `verifyHubConfiguration.metadata`?

Reply:
It's very broken atm and fixing it is out of scope of this card I think - I'll bear it in mind when/if we fix it though